src/popup/pages/Send/index.js

```diff
@@ -425,7 +430,7 @@ class SendPage extends React.Component {
         />}
         <CustomInput
           value={this.state.feeAmount}
-          placeholder={"Fee Amount"}
+          placeholder={`Fee Amount (nano ${this.props.netConfig.currentSymbol})`}
```

Comment:
Maybe call this `Gwei` since it's more accessible terminology than nano?

Reply:
wei is a fraction of ETH though

Reply:
Yeah, let's go with nano ROSE / nano TEST. I recall we also considered nROSE in the past :slightly_smiling_face: